stage-0-proposals.md

```diff
@@ -10,11 +10,11 @@ Stage 0 proposals are either
 |   | [Defensible Classes](http://wiki.ecmascript.org/doku.php?id=strawman:defensible_classes)                                                 | Mark Miller & Doug Crockford    | 0     |
 |   | [Relationships](http://wiki.ecmascript.org/doku.php?id=strawman:relationships)                                                           | Mark Miller & Waldemar Horwat   | 0     |
 | 🚀 | [`String.prototype.at`](https://github.com/mathiasbynens/String.prototype.at)                                                            | Mathias Bynens & Rick Waldron   | 0     |
-|   | [Structured Clone](https://github.com/dslomov-chromium/ecmascript-structured-clone)                                                      | Dmitry Lomov                    | 0     |
+|   | [Structured Clone](https://github.com/dslomov-chromium/ecmascript-structured-clone)                                                      | ?                               | 0     |
```

Comment:
I think it's better to leave Dmitry's name on it until it's withdrawn or until new champions have been found - the other change is good, thanks!